app/frontend/app/models/group-member.js

```diff
@@ -0,0 +1,25 @@
+import DS from 'ember-data';
+import Ember from 'ember';
+
+export default DS.Model.extend({
+  user: DS.belongsTo('user'),
+  rank: DS.attr('string'),
+  pending: DS.attr('boolean'),
+  groupId: DS.attr('string'),
+
+  // This suffers the same problem as `substories`, where the
+  // relationship only serializes a subset of `group-members`,
+  // rather than all and ember will destroy the relationship.
```

Comment:
It's not really a bug with Ember, just the way we have our API setup.

Because we only serialize a small amount of the members on the `GroupSerializer`, when we add more to that relation via visiting the `/members` page, Ember destroys all of those new relations when it gets that same `Group` data from another JSON payload.

There is probably a better way to handle it but this is the same hack I applied to substories (to fix the deletion bug).